samples/bluetooth/mesh/ble_peripheral_lbs_coex/README.rst

```diff
@@ -9,14 +9,19 @@ Bluetooth: Mesh and peripheral coexistence
 
 This sample demonstrates how to combine Bluetooth® mesh and Bluetooth® Low Energy features in a single application.
 
+FEM support
+***********
+
+.. include:: /includes/sample_fem_support.txt
+
```

Comment:
```suggestion
FEM support
===========

.. include:: /includes/sample_fem_support.txt

```
This should be moved as a subsection of Configuration section, right under Source file setup.
For all samples.